example/assets/styles/modal.css

```diff
@@ -103,17 +100,46 @@
       to { opacity: 1; }
 }
 
+@keyframes mmfadeOut {
+  from { opacity: 1; }
+  to { opacity: 0; }
+}
+
 @keyframes mmslideIn {
   from { transform: translateY(35%); }
     to { transform: translateY(0); }
 }
 
-.micromodal-slide[aria-hidden="false"] .modal__overlay {
-  animation: mmfadeIn .3s cubic-bezier(0.0, 0.0, 0.2, 1);
+@keyframes mmslideOut {
+  from { transform: translateY(0); }
+  to { transform: translateY(35%); }
+}
+
+.modal.is-open .modal__overlay {
+  animation: mmfadeIn .3s cubic-bezier(0, 0, 0.2, 1);
+}
+
+.modal.is-open .modal__container {
+  animation: mmslideIn .3s cubic-bezier(0, 0, 0.2, 1);
+}
+
+.modal.is-open {
+  transform: scale(1);
+}
+
+.modal.is-close {
```

Comment:
Do we need a specific `is-close` class? It should be the default right.

Reply:
It could have been default, but this class manages close animations. Without class, we can't put close animations stuff, as we don't know about modal state.